### PR TITLE
Set eviction-hard for vagrant cluster

### PR DESCRIPTION
**Pull request by dshulyak**
In order to test eviction related functionality it will be convenient to have reasonable eviction defaults.
At this moment exactly same flags are used by GCE environment

kubelet will have a following flag:

--eviction-hard=memory.available<100Mi,nodefs.available<10%

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/32066)

### CODE REVIEW

**Comment by dshulyak** [comment]
@derekwaynecarr do you think it is possible to land it this week? or better to wait?

File: cluster/vagrant/config-default.sh [hunk 1]
```diff
@@ -115,3 +115,6 @@ E2E_STORAGE_TEST_ENVIRONMENT=${KUBE_E2E_STORAGE_TEST_ENVIRONMENT:-false}
 
 # Default fallback NETWORK_IF_NAME, will be used in case when no 'VAGRANT-BEGIN' comments were defined in network-script
 export DEFAULT_NETWORK_IF_NAME="eth0"
+
+# Evict pods whenever compute resource availability on the nodes gets below a threshold.
+EVICTION_HARD="${EVICTION_HARD:-memory.available<100Mi,nodefs.available<10%}"
```
Comment (derekwaynecarr) [inline cluster/vagrant/config-default.sh]: nit: add a new line.

File: cluster/vagrant/config-test.sh [hunk 2]
```diff
@@ -26,3 +26,6 @@ REGISTER_MASTER_KUBELET=${REGISTER_MASTER:-false}
 
 # Optional: if set to true, kube-up will configure the cluster to run e2e tests.
 E2E_STORAGE_TEST_ENVIRONMENT=${KUBE_E2E_STORAGE_TEST_ENVIRONMENT:-false}
+
+# Evict pods whenever compute resource availability on the nodes gets below a threshold.
+EVICTION_HARD="${EVICTION_HARD:-memory.available<100Mi,nodefs.available<10%}"
```
Comment (derekwaynecarr) [inline cluster/vagrant/config-test.sh]: nit: add new line

**Comment by derekwaynecarr** [comment]
two nits, then this is good to go.  it does help testing 1.4 flakes, so i have no objection to getting this.

**Pull request merged**